PolyEngine/WindowsGLStandalone/Src/Main.cpp

```diff
@@ -5,6 +5,7 @@
 #include <Engine.hpp>
 #include <OpenGLRenderingContext.hpp>
 #include <TestGame.hpp>
+#include <string>
```

Comment:
Just used it for a temporary visualisation of delta time, so i didn't put much tought into it, but no problem i guess